rosdep/base.yaml

```
@@ -7160,6 +7160,19 @@ osmium:
   fedora: [libosmium-devel]
   nixos: [libosmium]
   ubuntu: [libosmium-dev]
+paho-mqtt-c:
+  alpine: [paho-mqtt-c]
+  debian:
+    bullseye: [libpaho-mqtt1.3]
+  fedora: [paho-c]
+  nixos: [paho-mqtt-c]
+  opensuse: [libpaho-mqtt1]
+  ubuntu:
+    jammy: [libpaho-mqtt1.3]
```

Here, we want this key to be forwards-compatible as much as possible.  So this should be:

```suggestion
  ubuntu:
    '*': [libpaho-mqtt1.3]
    focal: null
    bionic: null
```

(same goes for the C++ package below)